#! /usr/bin/env python
# Task Long configuration
import math

PACKAGE='task_manager_uavsim'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *
from task_manager_lib.parameter_generator import *

gen = TaskParameterGenerator()
#       Name                    Type            Description
#       Default    Min   Max
gen.add("acquisition_time",  double_t, 0,    "Time to wait on each sectors",  2., 0.0, 60.0)
gen.add("num_sectors",      int_t, 0,    "Number of angular step to scan",  6, 2, 30)

exit(gen.generate(PACKAGE, "task_manager_uavsim", "TaskRadioScan"))

